avocado/utils/cloudinit.py

```diff
@@ -174,21 +174,46 @@ def __init__(self, address, instance_id):
         self.instance_id = instance_id
         self.instance_phoned_back = False
 
+    def wait_for_phone_home(self, new_call=False):
+        """Waits for this instance to call.
+
+        :param new_call: Default is False, so if this instance was called back
+                         already, this method it would return immediately and
+                         will not wait for a new call.
```

Comment:
sorry, but I still do not properly understand this docstring. What the `it` means in `this method it would return immediately and will not wait for a new call`?

Shouldn't that sentence be: 
```
this method would return it immediately and will not wait for a new call
or
this method would return immediately and will not wait for a new call
```